org.openhab.binding.zigbee/README.md

```diff
@@ -121,6 +129,14 @@ Please note that, technically, you are not bound to using the values from the ta
 
 Some coordinators may need to allocate memory to handle each node in the network. This is an integer setting, and should be set to the maximum number of nodes expected to be added to the network. It should be noted that this will consume memory on the coordinator which may impact on other services such as packet buffers, so it is not advised to simply set this to the maximum value.
 
+**Values:**
```

Comment:
This is a `select` element in the configuration page.
![image](https://user-images.githubusercontent.com/551789/89524911-a9888300-d7e5-11ea-85e2-bb37df9b4214.png)

This change makes sense if the element stays as-is, otherwise I will also replace the `select` with an `input`

Reply:
This is an integer - the selection just gives you some preset values, but you can type in whatever you like. The way you have written this is confusing since someone defining their configuration in a text file will use the "value" of (for example) "large" and this will not work at all. You MUST use the number since it is an integer.

Reply:
> but you can type in whatever you like

For how much I try, I can't. But I see your point about config files. Will find another way to make the value of the select options more clear.

Reply:
Maybe this is an issue with the definition - I should check in that it might need another setting in the config file to allow other settings to be typed in. The intention is that it's an integer - well - it is an integer and it must be specified as an integer. The UI is just presenting this integer as a string.

Reply:
Made more clear by adding the actual value to the text of the `select` option